app/assets/javascripts/discourse/models/topic.js.es6

```diff
@@ -58,7 +59,13 @@ const Topic = RestModel.extend({
 
   @computed('fancy_title')
   fancyTitle(title) {
-    return censor(emojiUnescape(title || ""), Discourse.Site.currentProp('censored_words'));
+    let fancyTitle = censor(emojiUnescape(title || ""), Discourse.Site.currentProp('censored_words'));
```

Comment:
It wasn't in here before, but can you try `this.site` instead of `Discourse.Site` here? If it works it would be a win.

Reply:
It doesn't work. this.site isn't defined for the categories route.

Reply:
Ugh, I should fix that at some point.